pkg/version/windows_versioninfo.json

```diff
@@ -0,0 +1,44 @@
+{
```

Comment:
Right, this file has to be generated. 

To generate it, @shiywang you should be able to get version information from env vars like `OS_GIT_VERSION`, `OS_GIT_MAJOR` (Major), `OS_GIT_MINOR` (Minor), `OS_GIT_COMMIT` (Patch). Date modified is just something like `"$(date -u +'%Y-%m-%dT%H:%M:%SZ')"` (formatted the way `goversioninfo` expects it). 

Company Name, Product Name and Copyright can probably be static, unless we have it in a var somewhere (@stevekuznetsov clue?) or it differs for any of the products we build (Origin, OCP, Online). @brenton or @jupierce can you confirm this?